gensim/models/doc2vec.py

```diff
@@ -1171,13 +1177,18 @@ def _tag_seen(self, index, docvecs):
 
 class Doc2VecTrainables(Word2VecTrainables):
     """Represents the inner shallow neural network used to train :class:`~gensim.models.doc2vec.Doc2Vec`."""
-    def __init__(self, dm=1, dm_concat=0, dm_tag_count=1, vector_size=100, seed=1, hashfxn=hash, window=5):
+    def __init__(self, pretrained_emb, dm=1, dm_concat=0, dm_tag_count=1, vector_size=100, seed=1, hashfxn=hash, window=5):
+
+        self.pretrained_emb=pretrained_emb
```

Comment:
PEP8, here and everywhere

Reply:
How to change this with PEP8? Thank you for advice!